daemon/config/config.go

```diff
@@ -320,7 +320,10 @@ func getConflictFreeConfiguration(configFile string, flags *pflag.FlagSet) (*Con
 			return nil, err
 		}
 
-		configSet := configValuesSet(jsonConfig)
+		configSet, unnecessaryKeys := configValuesSet(jsonConfig)
+		if len(unnecessaryKeys) > 0 {
+			logrus.Warnf("The following configuration keys are unnecessary: %s", strings.Join(unnecessaryKeys, ","))
```

Comment:
Wondering how much value this adds. Having empty options in the configuration file may actually be a deliberate choice; similar to having "commented out" options in an ini-file. If these options are not changing the daemon's behaviour, I think it's fine to skip them without warning.

I _do_ think options that are _unknown_ should output a warning (thinking of the "typo" situation), but as commented earlier, I'd prefer to have a json-schema (or similar) approach for this.

I'm -1 on this change (sorry! 😅 )